plugins/yara/src/plugin.cpp

```diff
@@ -606,26 +627,54 @@ class plugin final : public virtual operator_plugin<yara_operator> {
     return {.transformation = true};
   }
 
+  auto make(invocation inv, session ctx) const
+    -> failure_or<operator_ptr> override {
+    auto args = operator_args{};
+    argument_parser2::operator_("yara")
+      // XXX: How do I get a variable number of args? std::vector<std::string>?
+      // .add(args.rules, "<rules>")
```

Comment:
I had a similar problem for `api` yesterday, see https://github.com/tenzir/tenzir/pull/4538#discussion_r1733517343

Can you write a follow-up issue for positional variadic arguments in `argument_parser2` and list down all the places you find where it'd be nice to use them, and then for now use a list as an argument here?